### DISCUSSION TITLE
Audio Feature

### DISCUSSION THREAD

**Discussion by kgschoueri** [Smithsonian/dpo-voyager#162] [General]
Hey Voyager Team. I have been able to get the newest version of Voyager (V0.21.1) where I can input an audio element and play it but I don't know how to use or configure it within a scene or tour. I saw you have a new menu icon for audio in the baseball helmet example. 

I tried to edit a tour with by playing the audio element but it also wasn't working. Also, what does the 'IsNarration' setting mean? 

We're really excited about the audio feature I just need some help understanding how to use it! 

Kelly

**Comment by gjcope** [comment]
Hi @kgschoueri - FYI we've made some bug fixes to the audio drag/drop zone and it should be much easier to use in the new release. Audio is an emerging feature and currently we are supporting it's use as a narration for the 3d scene. The narration icon and supporting feature will only work if the 'IsNarration' flag is enabled for an audio clip. This flag is in anticipation of using included audio in various other ways (such as audio triggered by object interaction) as well. Currently an included 'non-narration' clip will just be added to the scene package. We welcome ideas and proposals for other audio uses to support!

**Reply by kgschoueri** [reply]
@gjcope then I think there is a bug for the 'IsNarration' setting for V0.21.1. I load an audio, select IsNarration (Yes), save workspace, reload page and when I check that IsNarration is still selected it says (No). I will open an issue for this. Anyway, I will see if we can update to the newest release when it becomes available. 

For other audio uses. Being able to add multiple audio elements and then trigger them when creating steps in a tour would be cool! But also a way for users to disable the audio if they want.

**Reply by gjcope** [reply]
@kgschoueri In Voyager Story we currently have two different steps, saving ("capturing") the scene state and saving the scene package. The way narration works right now is that it is tied to the scene state. If you capture the state before saving, you should see it work as expected. This does still seem like a conceptual bug though. We can revisit trying to move that functionality to the package side.

**Reply by kgschoueri** [reply]
@gjcope Yes, I was able to get the IsNarration to say 'Yes' after a page reload by capturing the scene state. I would never had thought to do that since I thought the capture tool was just for capturing the position of the 3D model. However, despite it saying "Yes" I still don't see the audio icon in the menu bar...

**Reply by kgschoueri** [reply]
@gjcope I just tried this capture audio narration procedure with the older version we were working with and in this version it does add the audio icon but the issue with this version is that the audio itself isn't playing... I am somehow stuck between two versions that do one of the two things
